src/Enums/StatusEnum.php

```diff
@@ -6,7 +6,7 @@ enum StatusEnum: string
 {
     case LOADED = 'Already loaded';
     case IN_CACHE = 'Already in cache';
-    case INTERNALIZED = 'Internalized';
+    case CACHED = 'Cached';
     case INVALID = 'Not in a CDN or local filesystem, falling back to provided path';
     case DISABLED = 'Development mode active';
```

Comment:
Changing the enum is a BREAKING CHANGE. I vaguely remember using this somewhere... In backpack maybe? Idk. But it was like... if the response from basset is IN_CACHE then echo whatever.

If we change the enum... that response will change...

So it's a breaking change.